Uyanda.Coffee.WebApi/Controllers/BeveragesController.cs

```diff
@@ -15,9 +16,13 @@ public class BeveragesController : ControllerBase
     {
         private readonly IBeverageManagementService beverageManagementService;
 
-        public BeveragesController(IBeverageManagementService beverageManagementService)
+        private readonly IConfiguration configuration;   
```

Comment:
This field is not being used. Please don't keep unused code.

Reply:
field removed